hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -343,6 +343,11 @@
   public static final double
       HDDS_DATANODE_STORAGE_UTILIZATION_CRITICAL_THRESHOLD_DEFAULT = 0.95;
 
+  public static final String
+      HDDS_DATANODE_METADATA_CACHE_SIZE = "hdds.datanode.metadata.cache.size";
+  public static final String
+      HDDS_DATANODE_CACHE_SIZE_DEFAULT = "64MB";
```

Comment:
The usual pattern here is to have a pair of variables name like:

<PARAMETER>_KEY
<PARAMETER>_DEFAULT

I think it would be good to change this variable to HDDS_DATANODE_METADATA_CACHE_SIZE_DEFAULT.

I see some other parameters in this file don't end in "_KEY", so I guess it is not essential to rename HDDS_DATANODE_METADATA_CACHE_SIZE to "HDDS_DATANODE_METADATA_CACHE_SIZE_KEY".

If we change the parameter name to "hdds.datanode.container.rocksdb.cache.size" based on the above comment, maybe we should also rename the variables to HDDS_DATANODE_CONTAINER_ROCKSDB_CACHE_SIZE ?

Reply:
Yes, this was a typo. I forgot to update the name of the default variable when I updated the name. I will fix it once we decide on a name for the config.